### DISCUSSION TITLE
From fastq files to signac analysis

### DISCUSSION THREAD

**Discussion by Melody5528** [stuart-lab/signac#983] [Q&A] [answered]
Hi, I am trying to follow the PBMC tutorial for the scATAC-seq analysis. I have eight fastq files for each sample: 2 x R1, R2, R3, and I1.

The four files used in the tutorial: Raw data, metadata, fragment file, and fragment file index, seem very different than what I have.

How do I go from the fastq files to produce the required files for the workflow?

Thank you very much for the advice!!

**Comment by timoast** [comment]
There's an example here on the sinto website: https://timoast.github.io/sinto/scatac.html

Alternatively if you have 10x data you can run cellranger-atac

**Comment by Melody5528** [comment]
Thank you very much for your reply. 
My data is from 10X.
Is cellranger-atac the only pipeline to process the 10X data?
Could I convert them into signac compatible files? 
Many thanks again.







    On Friday, February 18, 2022, 03:43:13 PM EST, Tim Stuart ***@***.***> wrote:  
 
 


There's an example here on the sinto website: https://timoast.github.io/sinto/scatac.html

Alternatively if you have 10x data you can run cellranger-atac

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Melody5528** [comment]
Here are the possible output files from the Cellranger. Which ones correspond to the 4 files specified in the  Signac tutorial? Can I use them directly in signac or more modifications are needed? Any examples for modifications? Many thanks,
 \- Per-barcode fragment counts & metrics:        /home/jdoe/runs/sample345/outs/singlecell.csv
\- Position sorted BAM file:                     /home/jdoe/runs/sample345/outs/possorted_bam.bam
\- Position sorted BAM index:                    /home/jdoe/runs/sample345/outs/possorted_bam.bam.bai
\- Summary of all data metrics:                  /home/jdoe/runs/sample345/outs/summary.json
\- HTML file summarizing data & analysis:        /home/jdoe/runs/sample345/outs/web_summary.html
\- Bed file of all called peak locations:        /home/jdoe/runs/sample345/outs/peaks.bed
\- Raw peak barcode matrix in hdf5 format:       /home/jdoe/runs/sample345/outs/raw_peak_bc_matrix.h5
\- Raw peak barcode matrix in mex format:        /home/jdoe/runs/sample345/outs/raw_peak_bc_matrix
\- Directory of analysis files:                  /home/jdoe/runs/sample345/outs/analysis
\- Filtered peak barcode matrix in hdf5 format:  /home/jdoe/runs/sample345/outs/filtered_peak_bc_matrix.h5
\- Filtered peak barcode matrix in mex format:   /home/jdoe/runs/sample345/outs/filtered_peak_bc_matrix
\- Barcoded and aligned fragment file:           /home/jdoe/runs/sample345/outs/fragments.tsv.gz
\- Fragment file index:                          /home/jdoe/runs/sample345/outs/fragments.tsv.gz.tbi
\- Filtered tf barcode matrix in hdf5 format:    /home/jdoe/runs/sample345/outs/filtered_tf_bc_matrix.h5
\- Filtered tf barcode matrix in mex format:     /home/jdoe/runs/sample345/outs/filtered_tf_bc_matrix
\- Loupe Browser input file:                     /home/jdoe/runs/sample345/outs/cloupe.cloupe
\- csv summarizing important metrics and values: /home/jdoe/runs/sample345/outs/summary.csv
\- Annotation of peaks with genes:               /home/jdoe/runs/sample345/outs/peak_annotation.tsv
\- Peak-motif associations:                      /home/jdoe/runs/sample345/outs/peak_motif_mapping.bed


    On Friday, February 18, 2022, 03:43:13 PM EST, Tim Stuart ***@***.***> wrote:  
 
 


There's an example here on the sinto website: https://timoast.github.io/sinto/scatac.html

Alternatively if you have 10x data you can run cellranger-atac

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by timoast** [comment, marked as answer]
Most of the Signac vignettes start with files that were produced with cellranger-atac. See here for an example: https://satijalab.org/signac/articles/pbmc_vignette.html
